### PR TITLE
(#5) Cross platform

### PR DESCRIPTION
**Pull request by MikaelSmith**

### CODE REVIEW

**Comment by peterhuene** [comment]
Needs a rebase with the last merge.  Also, does the `#5` in the commit message mean something?

**Comment by MikaelSmith** [comment]
It links to an issue and should cause it to automatically close on merge.

File: 01-Getting-Started.md [hunk 1]
```diff
@@ -160,3 +173,129 @@ Vanagon contains the `build` command for generating a package. It also contains
...
+setup_logging(boost::nowide::cerr);
+boost::nowide::cout << "Hello!" << std::endl;
+
+// Use Boost.Nowide cin to correctly read UTF-8 characters.
+// On Windows this reads using a UTF-16 system call and converts to UTF-8.
+std::string input;
+boost::nowide::cin >> input;
+```
+
+Boost.Nowide's cout/cerr implementations come with the caveat that they aren't header-only, so if statically linking against Boost.Nowide flushing in different libraries will not be synchronized or affect text written in other libraries.
```
Comment (peterhuene) [inline 01-Getting-Started.md]: I think this reads a little oddly: `...so if statically linking against Boost.Nowide flushing in different libraries will not be synchronized...`.

Perhaps: `...aren't header-only and input/output performed by third party libraries will not be synchronized.`?

**Comment by peterhuene** [comment]
Other than above :+1:

File: 01-Getting-Started.md (hunk 1 above)
Comment (MikaelSmith) [inline 01-Getting-Started.md]: I'm trying to explain that if you statically link libboost_nowide into two binaries (a DLL and executable, for example), I/O operations in each of those binaries will not be synchronized with the other. Each needs to explicitly flush to ensure correct output ordering.

Comment (peterhuene) [inline 01-Getting-Started.md]: I see, perhaps `...so if Boost.Nowide is statically linked into multiple shared libraries, input and output must be manually synchronized between those libraries`?

Comment (MikaelSmith) [inline 01-Getting-Started.md]: :+1:

**Comment by MikaelSmith** [comment]
Updated.

**Pull request merged**